python/tests/python_test_raw.py

```diff
@@ -20,19 +20,19 @@ def propToCArray( L, prop, cArrayElType ):
 
 
 # load data
-print "--- Loading data ---"
+print("--- Loading data ---")
```

Comment:
Might be worth adding `from __future__ import print_function` as the first line above to make sure this behavior is maintained even on Python 2.